components/map/overview.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers. It is supported for both desktop and mobile devices.
+
+The component also provides tile layers, shape (vector) layers, and marker layers.
```

Comment:
The bubble layer can be mentioned as well - it is a special type of shape layer, but still, it is not incorrect to list it as a separate type of layer.

Reply:
Done